src/Modal/Modal.test.jsx

```diff
@@ -128,6 +129,27 @@ describe('<Modal />', () => {
       expect(wrapper.find('button')).toHaveLength(1);
     });
 
+    it('renders custom close button string', () => {
+      wrapper = mount(<Modal {...defaultProps} closeText={closeText} />);
+      const modalFooter = wrapper.find('.modal-footer');
+      const closeButton = modalFooter.find('button');
+
+      expect(closeButton).toHaveLength(1);
+      expect(closeButton.children()).toHaveLength(0);
+      expect(closeButton.text()).toEqual(closeText);
+    });
+
+    it('renders custom close button element', () => {
+      const closeElem = <span>{closeText}</span>;
+      wrapper = mount(<Modal {...defaultProps} closeText={closeElem} />);
+      const modalFooter = wrapper.find('.modal-footer');
+      const closeButton = modalFooter.find('button');
+
+      expect(closeButton).toHaveLength(1);
+      expect(closeButton.find('span')).toHaveLength(1);
```

Comment:
Should there also be a check that the `children` size should also be `1`? This guarantees that the `span` would be the only `child`.